src/panels/lovelace/common/create-card-element.js

```diff
@@ -69,7 +69,7 @@ function _createErrorElement(error, config) {
   return _createElement("hui-error-card", createErrorCardConfig(error, config));
 }
 
-export default function createCardElement(config) {
+export default function createCardElement(config, hass = undefined) {
```

Comment:
I think that this is not a good addition. It's conflating concerns. Config is standalone of `hass`.